packages/hoppscotch-app/components/lenses/renderers/composables/useDownloadResponse.ts

```diff
@@ -0,0 +1,35 @@
+import { Ref, ref } from "@nuxtjs/composition-api"
+
+export type downloadResponseReturnType = (() => void) | Ref<any>
+
+export default function useDownloadResponse(
+  contentType: string,
+  responseBodyText: Ref<any>,
+  toast: any,
+  t: any
+): { [key: string]: downloadResponseReturnType } {
```

Comment:
The return type defined is too broad, either narrow the return type remove the return type to let TypeScript figure it out.